var/spack/repos/builtin/packages/py-beancount/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBeancount(PythonPackage):
+    """A double-entry bookkeeping computer language that lets you define
+       financial transaction records in a text file, read them in memory,
+       generate a variety of reports from them, and provides a web
+       interface.."""
+
+    homepage = "http://furius.ca/beancount/"
+    pypi     = "https://pypi.io/packages/source/b/beancount/beancount-2.3.3.tar.gz"
+
+    version('2.3.3', sha256='d9a29839ea867d1dda7af1f4bf5d3959aa7c1574cd4a0bc86f69ee64c555c71c')
+
+    depends_on('python@3.5:',        type=('build', 'run'))
+    depends_on('py-setuptools',      type=('build'))
+    depends_on('py-pytest',          type=('test', 'run'))
```

Comment:
Is this actually needed at run-time? I think this is in `install_requires` by accident. We no longer run tests, so `type=test` isn't needed, but it might be needed at build-time to prevent setuptools from downloading anything

Reply:
Did you confirm that it isn't needed at build-time? Just want to make sure setuptools doesn't automatically download it.

Reply:
I just confirmed that the package build in its current state. I think that this must have been a remnant from when I tried to build the package from source.

Reply:
You should check the build log to see if pytest is downloaded and installed by setuptools. We want the package to build on air-gapped networks where missing deps will cause the build to fail.

Reply:
I don't see anything in the build logs that suggest that `pytest` is downloaded. I've attached the build log if you wish to double check.
[spack-build-out.txt](https://github.com/spack/spack/files/5900406/spack-build-out.txt)